core/common/src/main/java/alluxio/util/webui/NodeInfo.java

```diff
@@ -26,7 +28,9 @@
   private final String mLastContactSec;
   private final String mWorkerState;
   private final long mCapacityBytes;
+  private final String mCapacity;
   private final long mUsedBytes;
+  private final String mUsedMemory;
```

Comment:
Are the string and long form both necessary?

Reply:
no, removed one